projects/max14906/Makefile

```diff
@@ -0,0 +1,9 @@
+# Select the example you want to enable by choosing y for enabling and n for disabling
+DUMMY_EXAMPLE = n
```

Comment:
the word `dummy` is not fit, please rename to `basic` or something else

Reply:
@RaduSabau1 this is not resolved.